.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         id: build-clone-main
         run: |
           git clone --filter=blob:none --branch main https://github.com/cisco/ChezScheme.git
-          echo "HASH_KEY=$(cat ChezScheme/.git/refs/heads/main)" >> "${GITHUB_OUTPUT}"
+          echo "HASH_KEY=2$(cat ChezScheme/.git/refs/heads/main)" >> "${GITHUB_OUTPUT}"
```

Comment:
Is this `2` a typo, or does it do something I don't know about. Trying it locally seems to just prepend `2` to the hash which doesn't seem correct.

Reply:
Not, it's important, because without it, the current cache won't have arm64osx in it.

Reply:
It seems the `2` is essentially marking a "schema change" for the cache. What do you think about changing the `2` to something like `v2_` to help us recognize this in the future?

(When I saw a `2` before process substitution, I initially wondered if you were doing something novel with stderr.)

Reply:
Great idea, done!